Add inactivity_policy.md (#13)
neoforged/governance: inactivity_policy.md

# Inactivity Policy

> Like any other large projects, members on the NeoForge team may become inactive and stop contributing to the project over long periods of time. This document outline a policy to make sure that inactive members are routinely removed from the team in order to keep the team size at a healthy amount and fully active.

Reviewer: ```suggestion
Like in any other large project, members on the NeoForged team may become inactive and stop contributing to the project over long periods of time. This document outlines a policy to make sure that inactive members are routinely removed from the team in order to keep the team size at a healthy amount and fully active.
```

### What this document is

This document is:

>  * Intended to outline the procedure to prunning inactive Neoforged team members with minimal disagreements.

Reviewer: ```suggestion
 * Intended to outline the procedure to pruning inactive Neoforged team members with minimal disagreements.
```

### What this document is not

This document is not:
 * Intended to be used to remove disliked members.
 * Intended to ignore people that may have life situations that get in the way of contributions.

## Procedure

> Every year, 2 months before the next planned Steering Council elections, the Steering Council members will review the list of all NeoForged team members. If a member has not been contributing for what their role is for over at least 12 months (Maintainers working on projects, Moderators watching conversations), the Steering Council will begin following through these series of steps:

Reviewer: ```suggestion
Every year, 2 months before the next planned Steering Council elections, the Steering Council members will review the list of all NeoForged team members. If a member has not been contributing for what their role is for over at least 12 months (Maintainers working on projects, Moderators watching conversations, subteam members working on their respective subprojects), the Steering Council will begin following through these series of steps:
```

 * Reach out to the member in question by DM and ask about their inactivity to the team. Wait at least 2 weeks for a response.
   * If no response, reach out to other members on the person's subteam to verify that the person is inactive in the role they were assigned.
 * If the member is still active and there is evidence of recent contribution presented, then the investigation is ended there and member remains on the team.

>  * If the member provides a reasonable excuse for their inactivity such as life situations getting in the way, they may be granted an extra year to remain on the team so they can have the time to get back into contributing once they are available to do so. This grace period must be agreed upon by 2/3rds of the Steering Council.

Reviewer: ```suggestion
 * If the member provides a reasonable excuse for their inactivity, such as life situations getting in the way, they may be granted an extra year to remain on the team so they can have the time to get back into contributing once they are available to do so. This grace period must be agreed upon by 2/3rds of the Steering Council.
```

 * If the member fails to provide a resonable excuse or failed to reply and no one has seen any recent contributions, then 2/3rds of the Steering Council must agree that the evidence shown means the member will be stripped from the team and given the Former Member role.

## Other things to note

If a Former Member later wishes to rejoin the team, they can provide an application for the role and they will be judged just like any other person applying to join the team. 

> Steering Council members cannot be put through this process as per the (voting.md)[voting.md] document, Steering Council are re-elected every year and thus, are routinely judged and replaced to ensure there are always active council members.

Reviewer: ```suggestion
Steering Council members cannot be put through this process as per the (voting.md)[voting.md] document. Steering Council are re-elected every year and thus are routinely judged and replaced to ensure there are always active council members.
```

# Inactivity Policy

Like in any other large project, members on the NeoForged team may become inactive and stop contributing to the project over long periods of time. This document outlines a policy to make sure that inactive members are routinely removed from the team in order to keep the team size at a healthy amount and fully active.

### What this document is

This document is:
 * Intended to outline the procedure to pruning inactive Neoforged team members with minimal disagreements.

### What this document is not

This document is not:
 * Intended to be used to remove disliked members.
 * Intended to ignore people that may have life situations that get in the way of contributions.

## Procedure

### Main Review

> Every year, 7 months before the next planned Steering Council elections, the Steering Council members will review the list of all NeoForged team members. If a member has not been contributing for what their role is for over at least 12 months (Maintainers working on projects, Moderators watching conversations, subteam members working on their respective subprojects), the Steering Council will begin following through these series of steps:

Reviewer 2: "for over at least" > "for over"

 * Reach out to the member in question and ask about their inactivity to the team. Wait at least 2 weeks for a response.

>    * If no response, reach out to other members on the person's subteam to verify that the person is inactive in the role they were assigned.

Reviewer 3: Do we even still have a concept of subteam?

Reviewer 3: It should be sufficient to reach out to other team members.
